### DISCUSSION TITLE
Would you guys be willing to switch the theme for documentation?

### DISCUSSION THREAD

**Discussion by baseplate-admin** [qtile/qtile#4548] [Ideas] [answered]
Hi,

I love the project. I was wondering if you folks would want to switch to a modern theme for documentation ?

Some suggestion:
* [`furo`](https://github.com/pradyunsg/furo)
* [`shibuya`](https://github.com/lepture/shibuya)

**Comment by jwijenbergh** [comment, marked as answer]
Thanks for the discussion thread. I just made PR #4593 to move to the furo theme

**Comment by jwijenbergh** [comment]
Current candidates here: 
Furo theme: https://qtile--4593.org.readthedocs.build/en/4593/
Shibuya theme: https://qtile--4595.org.readthedocs.build/en/4595/

Scrollbar on Shibuya seems a bit broken on firefox but I can fix that (due to version selection overflowing)

**Reply by baseplate-admin** [reply]
Hi, is there any decision?

**Reply by jrwrigh** [reply]
If you're looking for an opinion, I like Furo.

**Reply by jwijenbergh** [reply]
More people like Shibuya better right now but furo is more closer to our current docs and has more contributions. No decision has been made yet
